tests/common/helpers/dut_utils.py

```diff
@@ -230,3 +232,30 @@ def decode_dut_and_container_name(name_str):
         container_name = name_list[0]
 
     return dut_name, container_name
+
+
+def verify_features_state(duthost):
+    """Checks whether the state of each feature is valid.
+
+    Args:
+      duthost: An Ansible object of DuT.
+
+    Returns:
+      If states of all features are valid, returns True; otherwise,
+      returns False.
+    """
+    feature_status, succeeded = duthost.get_feature_status()
+    if not succeeded:
+        logger.info("Failed to get list of feature names.")
+        return False
+
+    for feature_name, status in feature_status.items():
+        logger.info("The state of '{}' is '{}'.".format(feature_name, status))
+
+        if status not in ("enabled", "always_enabled", "disabled", "always_disabled"):
+            logger.info("The state of '{}' is invalid!".format(feature_name))
```

Comment:
please put more information in the log for debugging and troubleshoot, for example, what is the status now?

Reply:
The statement at line 253 will put the current `state` into the log.